File: Assets/Convergence Assets/Scripts/BallBump.js
static var minMag : float = 12.0;
static var bigMag : float = 60.0;

function OnCollisionEnter (collision : Collision) {
	var mag : float = collision.relativeVelocity.magnitude;
	if(mag > minMag)
	{
		if(mag > bigMag) mag = bigMag;
		GetComponent.<AudioSource>().volume = mag / bigMag;
		GetComponent.<AudioSource>().Play();
		//Debug.Log("collision " + mag);
	}
	//Debug.Log("Had a collision");
}

File: Assets/Convergence Assets/Scripts/BallBump.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class BallBump : MonoBehaviour
{
    public static float minMag;
    public static float bigMag;
    public virtual void OnCollisionEnter(Collision collision)//Debug.Log("Had a collision");
    {
        float mag = collision.relativeVelocity.magnitude;
        if (mag > BallBump.minMag)
        {
            if (mag > BallBump.bigMag)
            {
                mag = BallBump.bigMag;
            }
            this.GetComponent<AudioSource>().volume = mag / BallBump.bigMag;
            this.GetComponent<AudioSource>().Play();
        }
    }

    static BallBump()
    {
        BallBump.minMag = 12f;
        BallBump.bigMag = 60f;
    }

}